src/Workspaces/CSharpTest/Formatting/FormattingTests.cs

```diff
@@ -10288,6 +10330,75 @@ class C
 {
     new  (int, int) this[int i] { get => throw null; }
 }
+");
+        }
+
+        [Fact, Trait(Traits.Feature, Traits.Features.Formatting)]
+        [WorkItem(56543, "https://github.com/dotnet/roslyn/issues/56543")]
+        public async Task FormatAttributeOnLambda()
+        {
+            await AssertFormatAsync(
+                expected: @"
+var f = [Attribute] () => { };
+",
+                code: @"
+var f =  [Attribute] () => { };
+");
+        }
+
+        [Fact, Trait(Traits.Feature, Traits.Features.Formatting)]
+        [WorkItem(56543, "https://github.com/dotnet/roslyn/issues/56543")]
+        public async Task FormatAttributeOnLambda_TwoAttributes()
+        {
+            await AssertFormatAsync(
+                expected: @"
+var f = [Attribute][Attribute2] () => { };
+",
+                code: @"
+var f =  [Attribute]  [Attribute2] () => { };
+");
+        }
+
+        [Fact, Trait(Traits.Feature, Traits.Features.Formatting)]
+        [WorkItem(56543, "https://github.com/dotnet/roslyn/issues/56543")]
+        public async Task FormatAttributeOnMethod_TwoAttributes()
+        {
+            await AssertFormatAsync(
+                expected: @"
+[Attribute][Attribute2]
```

Comment:
❔  What did this do before your change? Just wondering if the output used to split them to their own lines (like `SpacingAfterAttribute_MultipleOnDeclaration` shows).

Reply:
Prior to this PR, this would leave the formatting as-is (ie. `[Attribute]  [Attribute2]` with 2 spaces)